src/components/app-content-renderer/virtual-assistant.js

```diff
@@ -0,0 +1,31 @@
+import { useFlag } from '@unleash/proxy-client-react';
+import React, { Fragment } from 'react';
+import { ScalprumComponent } from '@scalprum/react-core';
+
+import './virtual-assistant.scss';
+import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
+
+function VirtualAssistant() {
+  const showVirtualAssistant = useFlag(
+    'platform.landing-page.virtual-assistant'
+  );
+  const chrome = useChrome();
+
+  // Disable it for prod, any stable environment or if the feature flag is off
+  if (chrome.isProd() || !chrome.isBeta() || !showVirtualAssistant) {
```

Comment:
I would change it so it's not disabled completely on production
```suggestion
  if (!showVirtualAssistant || (chrome.isProd() && !chrome.isBeta())) {
```

This way when we move it to prod environment we can enable the feature flag and it will be available only on prod preview. Once we receive the GA date we can remove the or statement.

Reply:
We don't want to show this on stage-stable, and this will show it on stage-stable if the feature flag is true.

```
showVirtualAssistant = true;
isProd = false;
isBeta = false;

if (!showVirtualAssistant || (chrome.isProd() && !chrome.isBeta()))
=> if (!true || (false && !false)
=> if (false || (false && true))
=> if (false || false)
=> if (false) <-- This will skip the "hide" part and show the VA on stage-stable
```

If we want to show it on prod-beta and stage-beta only if the feature flag is enabled it should be:

```suggestion
  if ( !chrome.isBeta() || !showVirtualAssistant) {
```

or changing the order

```suggestion
  if (!showVirtualAssistant || !chrome.isBeta()) {
```

WDYT?

Reply:
but i see your point about not wanting to surface it to prod-stable when we enable it on stage-stable, but i guess we will add other feature flag or update this logic by that point.

Reply:
@karelhala Using your suggestion, but we will need this PR so it doesn't break in stage-stable:
https://github.com/RedHatInsights/chrome-service-backend/pull/336

Reply:
Let's go with rnabling this on stage stable as well. Stage is dev only so once this PR is merged it will automatically surface on both stage prev and stage stable.